### PR TITLE
feat: add new aavechan Aave/stkAave delegators group

### PR DESCRIPTION
**Pull request by MartinGbz**

### CODE REVIEW

File: group-generators/helpers/data-providers/big-query/big-query.ts [hunk 1]
```diff
@@ -252,18 +252,26 @@ export class BigQueryProvider {
       eventSignature,
       dataSet: dataUrl[this.network],
     });
+
+    console.log("cacheKey", cacheKey);
```
[the remark this answers is not in the document]
Comment (MartinGbz) [reply group-generators/helpers/data-providers/big-query/big-query.ts]: ahah already updated sir 🫡
(forgot to remove it just before sending the PR)

**Comment by alexandrupele** [comment]
maybe it's a good idea to also rename the group in the index.
`"aavechan-aave-delegates": aavechanAaveDelegators,`

**Comment by MartinGbz** [comment]
> maybe it's a good idea to also rename the group in generators/index.ts `"aavechan-aave-delegates": aavechanAaveDelegators,`

Yes good catch, forgot this one thanks!

File: group-generators/helpers/data-operators/map.ts [hunk 2]
```diff
@@ -1,15 +1,44 @@
...
 
+export type Thresholds = {
+  min: number;
+  newValue: number;
+}[]
+
 export const Map = (
   groupData: FetchedData,
-  value: BigNumberish
+  value?: BigNumberish,
```
Comment (alexandrupele) [inline group-generators/helpers/data-operators/map.ts:11]: i think this is a good feature to have 👍 
however, now the map interface is a bit unclear
i suggest we keep the old map as it was and create a new map that allows more complicated mapping schemes
something like this
```
enum MapCriteria {
  LT = 'LT',
  GT = 'GT',
  EQ = 'EQ',
}

type MapOptions = Array<{
  criteria: MapCriteria;
  newValue: number;
}>;

const options: MapOptions = [
  { criteria: MapCriteria.LT, newValue: 10 },
  { criteria: MapCriteria.GT, newValue: 20 },
  { criteria: MapCriteria.EQ, newValue: 30 },
];
```

**Review by alexandrupele — APPROVED** [review]
awesome!

**Review by yum0e — APPROVED** [review]

**Pull request merged**